### DISCUSSION TITLE
The PHP8 does not fully work on Android 13.

### DISCUSSION THREAD

**Discussion by ghost** [termux/termux-packages#19243] [Q&A]
I am using a PHP website program based on Laravel. But I couldn't login its backend. After I log in, the backend of this website will automatically logout in 3s. But Wordpress and Typecho works fine. In Android 10 or Android 7.1.2 ,the website program is ok. Why?

**Comment by sk0kanik** [comment]
from [README](https://github.com/termux/termux-app/blob/master/README.md):

> **NOTICE: Termux may be unstable on Android 12+.** Android OS will kill any (phantom) processes greater than 32 (limit is for all apps combined) and also kill any processes using excessive CPU. You may get `[Process completed (signal 9) - press Enter]` message in the terminal without actually exiting the shell process yourself. Check the related issue [#2366](https://github.com/termux/termux-app/issues/2366), [issue tracker](https://issuetracker.google.com/u/1/issues/205156966), [phantom cached and empty processes docs](https://github.com/agnostic-apollo/Android-Docs/blob/master/en/docs/apps/processes/phantom-cached-and-empty-processes.md) and [this TLDR comment](https://github.com/termux/termux-app/issues/2366#issuecomment-1237468220) on how to disable trimming of phantom and excessive cpu usage processes. A proper docs page will be added later. An option to disable the killing should be available in Android 12L or 13, so upgrade at your own risk if you are on Android 11, specially if you are not rooted.

**Reply by ghost** [reply]
I have disabled Phantom Process Killer, otherwise even Nginx cannot start. But this method seems to have no impact on PHP. I did not find any 502 and 500 errors. When I run the same program in kali linux from Anlinux，it works fine.

**Comment by ghost** [comment]
The reason has been found, but currently the bootloader cannot be unlocked and cannot be resolved. Due to Android 11's modification of /sdcard, the PHP program on sdcard cannot function properly. The only solution is download the GSI from https://github.com/phhusson/treble_experimentations/releases and flash it into your phone. Then open the settings and enable the Disable FUSE function. Then restart.
